src/main/kotlin/com/exactpro/th2/http/client/ClientOptions.kt

```diff
@@ -71,13 +71,13 @@ internal class ClientOptions(
     }
 
     override fun onRequest(httpRequest: RawHttpRequest): RawHttpRequest {
-        logger.info { "Sending request: $httpRequest" }
+        logger.debug { "Sending request: $httpRequest" }
```

Comment:
Currently we talk about performance test execution, in this case info log with the whole details will slow down performance

Reply:
But you can change the logging via configuration, right? For your 'performance' test execution. Why should all other users be affected by that?

Reply:
info logging is usual level. Are users interested in `Sending request` or `Received response` log records when conn works as expected?

Reply:
You can ask them) But I would be interested in that because once there is an issue with the conn you still can find some evidence in the logs

Reply:
I would agree if you split the logging into two parts:

1. Light logging in info level (HTTP method, request path, first 100 bytes from body, etc)
2. Full logging in debug/trace level with the whole HTTP request

But this all is just my comments. If you think that it would be better to have a debug level here, you can go ahead. I just wanted to point out that this change might affect other people

Reply:
Let's print uri only for info log, because we need to execute some operation to extract, cat and convert to string

Reply:
As I said, no objections from my side. Use what you think suites the best here